GPU/Workflow/src/GPUWorkflowSpec.cxx

```diff
@@ -532,7 +543,8 @@ DataProcessorSpec getGPURecoWorkflowSpec(gpuworkflow::CompletionPolicyData* poli
               if (verbosity) {
                 end = std::chrono::high_resolution_clock::now();
                 std::chrono::duration<double> elapsed_seconds = end - start;
-                LOG(INFO) << "Allocation time for " << name << " (" << size << " bytes)" << ": " << elapsed_seconds.count() << "s";
+                LOG(INFO) << "Allocation time for " << name << " (" << size << " bytes)"
+                          << ": " << elapsed_seconds.count() << "s";
```

Comment:
Please don't break my lines :)

Reply:
I think this was clang-format.

Reply:
haha, sorry. My auto format keeps breaking, and I keep forgetting to fix it.